### ISSUE TITLE
SecurityError should be exposed

### ISSUE THREAD

**Issue by luke-coastpay-com** [seriousme/fastify-openapi-glue#631]
Tl;dr: [SecurityError](https://github.com/seriousme/fastify-openapi-glue/blob/master/lib/securityHandlers.js#L3) should be exported from the package

Howdy!

My `SecurityHandler` will throw different errors based on what went wrong, and I'd like to be able to access `SecurityError.errors` in my `ErrorHandler`. Since the `ErrorHandler` does not know which variety of error it is handling, I'd like to be able to check whether a given error is an instance of a `SecurityError` thrown by `fastify-openapi-glue` before attempting to access properties of `SecurityError`.

Currently, `SecurityError` is [exported](https://github.com/seriousme/fastify-openapi-glue/blob/master/lib/securityHandlers.js#L3) from `securityHandler.js`, but is not exported from the package, so I cannot access it in my code. It would be great if this error could be exported from the package.

Thanks!

**Comment by seriousme** [comment]
Hi,

thanks for asking. 
I will add it to the exports.

Kind regards,
Hans

**Comment by luke-coastpay-com** [comment]
Thanks!

**Comment by seriousme** [comment]
Published as v4.8.0 on NPM, enjoy!
